mev-build-rs/src/service.rs

```diff
@@ -72,7 +72,13 @@ impl Service {
 
         let genesis_details = client.get_genesis_details().await?;
         let genesis_validators_root = genesis_details.genesis_validators_root;
-        let clock = context.clock(Some(genesis_details.genesis_time));
+        let clock = match context.clock() {
```

Comment:
this file ends up deleted after #112 so we can leave alone

Reply:
cool!